src/vs/base/browser/domSanitize.ts

```diff
@@ -304,21 +288,22 @@ function doSanitizeHtml(untrusted: string, config: DomSanitizerConfig | undefine
 
 		resolvedConfig.ALLOWED_ATTR = Array.from(allowedAttrNames);
 
-		store.add(hookDomPurifyHrefAndSrcSanitizer(
+		hookDomPurifyHrefAndSrcSanitizer(
 			{
 				override: config?.allowedLinkProtocols?.override ?? [Schemas.http, Schemas.https],
 				allowRelativePaths: config?.allowRelativeLinkPaths ?? false
 			},
 			{
 				override: config?.allowedMediaProtocols?.override ?? [Schemas.http, Schemas.https],
 				allowRelativePaths: config?.allowRelativeMediaPaths ?? false
-			}));
+			});
+
 		if (config?.replaceWithPlaintext) {
-			store.add(addDompurifyHook('uponSanitizeElement', replaceWithPlainTextHook));
+			dompurify.addHook('uponSanitizeElement', replaceWithPlainTextHook);
 		}
 
 		if (allowedAttrPredicates.size) {
-			store.add(addDompurifyHook('uponSanitizeAttribute', (node, e) => {
+			dompurify.addHook('uponSanitizeAttribute', (node, e) => {
```

Comment:
I think I remember doing this because otherwise it was too easy to forget to clean up hooks properly

Reply:
Maybe the difference was that we were removing them individually before and not using `removeAllHooks` and one was missed